Source/cmake/OptionsHaiku.cmake

```diff
@@ -77,6 +77,8 @@ WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_INPUT_TYPE_COLOR PUBLIC ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_INPUT_TYPE_DATE PUBLIC ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_INPUT_TYPE_MONTH PUBLIC ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_INPUT_TYPE_TIME PUBLIC ON)
+WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_KEYBOARD_CODE_ATTRIBUTE PUBLIC ON)
```

Comment:
This should be set in WTF/wtf/FeatureDefines.h as is done for GTK and WPE ports. It is not supposed to be configurable through cmake (since the haiku implementation doesn't check for it using #ifdef and will always be enabled, resulting in a broken build if one attempts to disable this)